gemfiles/rails42.gemfile

```diff
@@ -3,17 +3,20 @@
 source "https://rubygems.org"
 
 gem "administrate-field-image"
+gem "autoprefixer-rails"
 gem "bourbon", "~> 5.0.0.beta.7"
 gem "delayed_job_active_record"
 gem "faker"
 gem "high_voltage"
 gem "markdown-rails"
 gem "pg"
 gem "redcarpet"
+gem "sentry-raven"
 gem "unicorn"
 gem "rails", "~> 4.2.0"
 
 group :development do
+  gem "refills"
```

Comment:
I suggest removing Refills because it is going through a major rewrite. The gem is likely to go away (or dormant).

The gem is just a generator anyway, so we don’t _need_ to have it to use some of the components.